src/appleseed.cli/main.cpp

```diff
@@ -327,13 +327,62 @@ namespace
             params.insert("noise_seed", noise_seed);
         }
 
+        if (g_cl.m_output.is_set())
+        {
+            const char* file_path = g_cl.m_output.value().c_str();
+            params.insert("output_path", file_path);
+        }
+
+        if (g_cl.m_checkpoint_create.is_set())
+        {
+            params.insert("checkpoint_create", "on");
```

Comment:
We usually prefer letting the system convert the value to a string, hence the following is preferred:

    params.insert("checkpoint_create", true);